webview-ui/src/Kaito/KaitoModels.tsx

```diff
@@ -89,13 +89,20 @@ export function KaitoModels(initialState: InitialState) {
         return defaultMessage;
     }
 
-    function generateYAML(model: string) {
+    function generateYAML(model: string): [string, string | undefined] {
         const modelDetails = getModelDetails(model);
         const name = stringOrUndefined(modelDetails?.modelName);
         const gpu = stringOrUndefined(modelDetails?.minimumGpu);
         let appname = name;
         let metadataname = name;
         let inferencename = name;
+        let privateConfig = "";
+        if (name.substring(0, 7) === "llama-2") {
```

Comment:
💡 Let's definitely code comment things like this for future understanding, please. :)